exporter/collector.go

```diff
@@ -144,7 +145,7 @@ func extractValue(logger *slog.Logger, data []byte, path string, enableJSONOutpu
 		return res, nil
 	}
 
-	return buf.String(), nil
+	return strings.TrimSpace(buf.String()), nil
```

Comment:
Should we always trim? Or add a boolean flag for trimming whitespace. :thinking: